### ISSUE TITLE
Why the comprehensions not supported when define workloads by cue?

### ISSUE THREAD

**Issue by rapkey** [kubevela/kubevela#1784]
```
foo: 1
a: [
    "a",
    if foo == 1 {
        "b"
    }     
```

**Comment by rapkey** [comment]
@wonderflow

**Comment by leejanee** [comment]
cue 0.2.x version does not support this writing
Because some features used by vela are not yet supported in the higher version of cue

**Comment by rapkey** [comment]
@leejanee Got it. Will supported in the nearest version?

**Comment by wonderflow** [comment]
#1400  We're discussing with CUE team with this issue.

**Comment by wonderflow** [comment]
Hi, @rapkey It's really really sorry for the long time upgrade process, Finally we have managed to fix it. The CUE upgrade has been done in the master banch and will be released in the v1.6. Hope you're still interested in it.

Thanks!
